packages/api-explorer/src/utils/hooks/navigation.ts

```diff
@@ -76,7 +76,11 @@ export const useNavigation = () => {
    */
   const buildPathWithGlobalParams = (path: string) => {
     const params = new URLSearchParams(history.location.search)
-    params.delete('v')
+    for (const key of params.keys()) {
+      if (key !== 's' && key !== 'sdk') {
```

Comment:
store all global params in an array and check that array for validity. i.e this check becomes:
`if (!globalParams.includes(key) params.delete(key)`